File: eva-core/src/main/java/io/nerv/core/exception/FileUploadException.java
package io.nerv.core.exception;

import io.nerv.core.enums.BizCodeEnum;
import lombok.Getter;

/**
 * 图片上传异常
 */
@Getter
public class FileUploadException extends RuntimeException{
    private String msg;

    public FileUploadException(String msg) {
        super();
        this.msg = msg;
    }

    public FileUploadException(BizCodeEnum bizCodeEnum) {
        super();
        this.msg = bizCodeEnum.getName();
    }
}

File: eva-core/src/main/kotlin/io/nerv/exception/FileUploadException.kt
package io.nerv.exception

import io.nerv.core.enums.BizCodeEnum
import lombok.Getter

/**
 * 图片上传异常
 */
@Getter
class FileUploadException : RuntimeException {
    var msg: String

    constructor(msg: String) : super() {
        this.msg = msg
    }

    constructor(bizCodeEnum: BizCodeEnum) : super() {
        msg = bizCodeEnum.code
    }
}